scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -230,6 +237,31 @@ boolean pauseJob(JobId jobId) {
         }
     }
 
+    boolean updateStartAt(JobId jobId, String startAt) {
+        JobData jobData = lockJob(jobId);
+        if (jobData == null) {
+            return false;
+        }
+        try {
+            InternalJob job = jobData.job;
+
+            long scheduledTime = ISO8601DateUtil.toDate(startAt).getTime();
+
+            Set<TaskId> updatedTasks = job.updateStartAtAndTasksScheduledTime(startAt, scheduledTime);
+
+            if (updatedTasks.size() > 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1155)